app/questionnaire/routing/helpers.py

```diff
@@ -10,7 +10,7 @@ def _casefold(value: Union[list, ValueTypes]) -> Union[list, ValueTypes]:
     if isinstance(value, str):
         return value.casefold()
 
-    if isinstance(value, list):
+    if isinstance(value, (list, tuple)):
```

Comment:
As discussed, we should update this to check if it is a `Sequence` instead of adding `tuple`.

Reply:
Changed to Sequence now, thanks!